src/views/Login.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="container">
+    <img src="../assets/wolox-logo.png" class="logo">
+    <div class="input-container">
+      <label for="name" class="label">First name</label>
+      <input id="name" class="input" v-model="firstName">
+    </div>
+    <div class="input-container">
+      <label for="name" class="label">Last name</label>
+      <input id="name" class="input" v-model="lastName">
+    </div>
+    <div class="input-container">
+      <label for="name" class="label">Email</label>
+      <input id="name" class="input" v-model="email">
+    </div>
+    <div class="input-container">
+      <label for="name" class="label">Password</label>
+      <input id="name" class="input" v-model="password" type="password">
+    </div>
+    <button :class="'primaryButton'" @click="printForm">Sign up</button>
```

Comment:
this can be done without :class

Reply:
It is because the class is not going to change so the bind is not necessary, right?